pages/nyamatvarden.vue

```diff
@@ -69,7 +96,6 @@ export default {
   methods: {
     sendData () {
       const component = this
-      console.log(component)
       axios({
         method: 'POST',
         url: 'https://iotsundsvall.northeurope.cloudapp.azure.com/api/graphql',
```

Comment:
TODO: We should probably consider extracting this value into an environment variable as it probably will change in the future and we don't want to require a code update when that happens.